### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by JunPeras**
Creaste pùll request

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -2,13 +2,9 @@
...
     TESTING = False
-    MONGO_URI = ''
-    GATEWAY_URL = ''
     
 class HerokuConfig(Config):
-    if 'MONGO_URI' in os.environ:
-        MONGO_URI = os.environ["MONGO_URI"]
-    if 'GATEWAY_URL' in os.environ:
-        GATEWAY_URL = os.environ["GATEWAY_URL"]
+        MONGO_URI = 'mongodb+srv://MongoDB:N1mHcZvuzF8VJXLD@micro-courses-kuizu.1uhbtuv.mongodb.net/?retryWrites=true&w=majority'
```
Comment (Ingrid-E) [inline config.py]: Restore as it was

File: run.py [hunk 2]
```diff
@@ -4,5 +4,5 @@
 app = create_app()
 
 if __name__ == '__main__':
-    port = os.getenv('PORT', default=8000)
+    port = os.getenv('PORT', default=3001)
```
Comment (Ingrid-E) [inline run.py]: Restore

**Comment by JunPeras** [comment]
Actions fixed

**Comment by JunPeras** [comment]
Actions fixed

**Pull request merged**